OracleDatabase/SingleInstance/README.md

```diff
@@ -180,6 +184,49 @@ Archive mode can be enabled during the first time when database is created by se
 
 In case this parameter is set `true` and passed to `docker run` command while reusing existing datafiles, even though this parameter would be visible as set to `true` in the container environment, this would not be set inside the database. The value used at the time of database creation will be used.
 
+#### Configuring TCPS in the database (Support from version 19.3.0 onwards)
+There are two ways to enable TCPS in the database:
```

Comment:
There are two ways to enable TCPS connections for the database

Reply:
Fixed.